docs/source/release_notes.rst

```diff
@@ -6,6 +6,7 @@ Release Notes
 Future Release
 ==============
     * Enhancements
+        * Add validation to URL logical type (:pr:`1285`)
```

Comment:
Not related to this PR: I think we missed this last time, but the entry for validating email address should be in this Future Release section, and not the v0.12.0 release notes section, correct?

If so, can you please fix that?

Reply:
fixed